ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/LoggingTests.kt

```diff
@@ -510,6 +513,42 @@ class LoggingTest : ClientLoader() {
         }
     }
 
+    @Test
+    fun testBodyLoggingKeepsContent() = clientTests {
+        val logs = ConcurrentList<String>()
+        val testLogger = object : Logger {
+            override fun log(message: String) {
+                logs.add(message)
+            }
+        }
+
+        config {
+            Logging {
+                logger = testLogger
+                level = LogLevel.ALL
+            }
+        }
+
+        test { client ->
+            val response = client.post<HttpResponse>("$TEST_SERVER/content/echo") {
+                body = MultiPartFormDataContent(formData {
+                    append("file", "123")
+                })
+            }
+
+            assertNotNull(response.receive<String>())
+            val request = response.request
+            val contentLength = request.content.contentLength!!
+            val contentType = request.content.contentType!!
+
+            assertTrue(contentType.contentType == "multipart")
```

Comment:
I'm not sure if it's ok to use such checks for `logfile`. I would prefer to list the whole here, as in other logging tests.

Reply:
Unfortunately, it's not possible here. Headers are in the same line with `boundary`. This line is flaky, so we can't rely on `TestLogger`